### PR TITLE
Initial Check in

### PR DESCRIPTION
**Pull request by drawyan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# unflatten-recursive [![CircleCI](https://circleci.com/gh/UrbanCompass/unflatten-recursive.svg?style=shield)](https://circleci.com/gh/UrbanCompass/unflatten-recursive)
+
+Unflattens object recursively while preserving arrays. Contents of arrays are unflattened.
+
+Note: UNLIKE OTHER OPEN SOURCE LIB, this lib also looping through arrays and unflatten the array
```
Comment (onlywei) [inline README.md]: Should be "loops" (not "looping"), and "unflattens" (not "unflatten")

Comment (onlywei) [reply README.md]: Also we should always specify WHICH "other open source lib", not just say "UNLIKE OTHER"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  "keywords": [
+    "convert",
+    "conversion",
+    "string",
+    "start",
+    "case"
+  ],
+  "contributors": [
+    "Zhuoyuan Zhang <zhuoyuan@compass.com>"
```
Comment (onlywei) [inline package.json:29]: Nice, this one is 100% you!

**Review by onlywei — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (drawyan) [reply README.md]: Done

**Pull request merged**